PDFTron/pdftron-flutter
# [Fix|Android] Support follow system nightmode flag in widget viewer

## Test
1. Apply patch: 
[dark-mode-flutter.patch](https://github.com/PDFTron/pdftron-flutter/files/14351527/dark-mode-flutter.patch)
2. Enable system dark mode.
3. Run example and ensure the viewer is not in dark mode.
4. Test for both widget and plugin viewer.

Closes https://apryse.atlassian.net/browse/FLUSDK-28
